packages/protocol/contracts/bridge/Bridge.sol

```diff
@@ -286,7 +286,11 @@ contract Bridge is EssentialContract, IBridge {
                     // - need to have a buffer/small revenue to the realyer since it consumes
                     // maintenance and infra costs to operate
                     uint256 refund = stats.numCacheOps * _GAS_REFUND_PER_CACHE_OPERATION;
-                    stats.gasUsedInFeeCalc = uint32(GAS_OVERHEAD + gasStart - gasleft());
+                    // Taking into account the encoded message calldata cost, and can count with 16
```

Comment:
could you add a TODO in the L1 contract log file to upgrade the Bridge on both L1 and L2.